core/controllers/topics_and_skills_dashboard.py

```diff
@@ -140,6 +141,7 @@ def post(self):
         description = self.payload.get('description')
         linked_topic_ids = self.payload.get('linked_topic_ids')
         explanation_dict = self.payload.get('explanation_dict')
+        temp_skill_id = self.payload.get('temp_skill_id')
```

Comment:
Can we not call this a "temp skill ID"? It has a lot of additional connotations if you do that. How about "temp_folder_name_for_images" or something like that?

We should also ensure that these temporary IDs use a different charset from the regular ones so that there is no possibility of overlap. Perhaps have a temp_[0-9]{8} format ("temp_" followed by 8 digits). Additionally can we have the images go in a /temp subfolder of the skill images directory, rather than be mixed in with the other existing skills?

Reply:
CHanged implementation